packages/core/src/queries/user.ts

```diff
@@ -158,3 +158,15 @@ export const hasActiveUsers = async () =>
     from ${table}
     limit 1
   `);
+
+export const getDailyNewUserCountsByTimeInterval = async (
+  startTimeExclusive: number,
+  endTimeInclusive: number
+) =>
+  envSet.pool.any<{ date: string; count: number }>(sql`
+    select date(${fields.createdAt}), count(*)
+    from ${table}
+    where ${fields.createdAt} > to_timestamp(${startTimeExclusive}::double precision / 1000)
+    and ${fields.createdAt} <= to_timestamp(${endTimeInclusive}::double precision / 1000)
+    group by date(${fields.createdAt})
+  `);
```

Comment:
Should this method be placed at `packages/core/src/lib/dashboard.ts` since it is in the 'dashboard' scope?

Reply:
Do we need to filter those users whose `create_at` is null?

Reply:
created_at is `not null`.

Reply:
It's better to put SQL queries in `queries/*.ts` grouping by the table.